geyser-plugin-interface/src/geyser_plugin_interface.rs

```diff
@@ -157,6 +158,26 @@ pub struct ReplicaTransactionInfoV2<'a> {
     pub index: usize,
 }
 
+/// Information about a transaction, including index in block
+#[derive(Clone, Debug)]
+#[repr(C)]
+pub struct ReplicaTransactionInfoV3<'a> {
+    /// The transaction message hash, used for identifying the transaction.
+    pub message_hash: &'a Hash,
```

Comment:
I changed this from `signature` to `message_hash` because the message hash used to only be available from the `SanitizedTransaction` and because you can get the tx signature from the `VersionedTransaction`